Assets/MixedRealityToolkit.Tests/TestUtilities.cs

```diff
@@ -25,6 +26,7 @@ public static class TestUtilities
         const string additiveTestSceneTemporarySavePath = "Assets/__temp_additive_test_scene_#.unity";
         public static Scene primaryTestScene;
         public static Scene[] additiveTestScenes = new Scene[0];
+        public static Boolean DisableGazeStabilization = true;
```

Comment:
What is this for?

Reply:
Ah, this is an artifact from some earlier testing I was doing, where I found that in test cases a gaze stabilizer was also being created (which it does in production, but for some reason in today's test cases, it isn't getting created). This led to some weirdness in certain tests (i.e. HoloLens1 based gaze tests, where it took N frames from gaze to stabilize on the right point).

I added this then to fix some of those tests but since then this isn't necessary, so the changes to this file can be deleted.